cmd/crossplane/core/init.go

```diff
@@ -90,7 +93,10 @@ func (c *initCommand) Run(s *runtime.Scheme, log logging.Logger) error {
 
 	steps = append(steps, initializer.NewLockObject(),
 		initializer.NewPackageInstaller(c.Providers, c.Configurations),
-		initializer.NewStoreConfigObject(c.Namespace))
+		initializer.NewStoreConfigObject(c.Namespace),
+		initializer.NewTLSCertificateGenerator(c.Namespace, c.TLSCASecretName, c.TLSServerSecretName, c.TLSClientSecretName, "crossplane", nil, initializer.TLSCertificateGeneratorWithLogger(log.WithValues("Step", "TLSCertificateGenerator"))),
```

Comment:
What about moving the `nil` owners list here to a `TLSCertificateGeneratorOption`?

Reply:
I agree with @phisco - six arguments plus variadic options does seem like a lot. If owner refs aren't always used it makes sense to me that they should be an option rather than a fixed argument.

Reply:
made it an option